x-pack/plugins/ml/public/components/controls/select_severity/select_severity.js

```diff
@@ -49,13 +49,19 @@ function optionValueToThreshold(value) {
   return threshold;
 }
 
+// This service will be populated by the corresponding angularjs based one.
+export const mlSelectSeverityService = {
+  state: null
+};
+
 class SelectSeverity extends Component {
   constructor(props) {
     super(props);
 
     // Restore the threshold from the state, or default to warning.
-    if (this.props.mlSelectSeverityService) {
-      this.mlSelectSeverityService = this.props.mlSelectSeverityService;
+    if (
+      typeof mlSelectSeverityService.state.get === 'function') {
```

Comment:
Nit - if conditional on a new line (unless the line length was too long?)

Reply:
Thanks, I changed this whole check a bit in 694ff95f79.